### ISSUE TITLE
Behaviour of lice on dead fish

### ISSUE THREAD

**Issue by erolm-a** [resistance-modelling/slim#38]
As discussed offline dead fish should likely kill the lice attached to them or at least force them to detach with a certain probability. As outlined by others, this probability is influenced by many factors such as sex, lifestage, or even stress factors.

To model that the model should likely:
- keep a statistical distribution of lice stage ages attached to fish (which should be the same of the normal lice population in principle?)
- determine some "dropout" coefficient based on sex and stage (I'd guess `0.5*p*(stages[L4:L4+2]) `, but I am not sure if female PA and A lice can detach if they have not mated yet)
- other ideas?

**Comment by anthonyohare** [comment]
Armin's thoughts:

Your question [what happens to the sea lice that are attached to a fish when the fish dies? Do the lice stay attached and eventually die or do they detach and look for another host.] is a difficult one. Depending on stage and sex, lice can move between hosts.
After settlement on the fish, the first stages, i.e. the infectious copepodid and the two following chalimus stages are tethered to the skin of the fish. Therefore these stages cannot move from one host fish to the next.

Next in the life cycle are two pre-adult stages and the final adult stage. These can move freely over the body surface of the fish, and in principle move away from the fish. The degree to which they do that has not been assessed by many studies, but it has been found that, due to their reproductive strategy, males tend to try and move to another host fish once all females on their given host have mated. 
Stephenson, J. F. (2012). The chemical cues of male sea lice Lepeophtheirus salmonis encourage others to move between host Atlantic salmon Salmo salar. Journal of Fish Biology, 81(3), 1118–1123. https://doi.org/10.1111/j.1095-8649.2012.03347.x

Reproductive strategy: Females typically mate only once and store the spermatophores from that mating to fertilise all batches eggs produced over their life time, but genetic paternity analysis has been shown that a significant number of females have offspring derived from more than one male, suggesting either they mate again after loss of spermatophores, or in the typical mating time window at the beginning of their reproductive phase mate with more than one male. ) Another factor contributing to the tendency of adult females to stay on the same host is that they are larger and  swim less well compared to males. 
Todd, C. D., Stevenson, R. J., Reinardy, H., & Ritchie, M. G. (2005). Polyandry in the ectoparasitic copepod Lepeophtheirus salmonis despite complex precopulatory and postcopulatory mate-guarding. Marine Ecology Progress Series, 303, 225–234.

Finally, I would expect that lice try to move to other hist fish if the fish is very sick --- lice seem to be able to sense that. Our technicians report that when there is a new batch of host fish that are still stressed due to tank transfers and do not feed, infections are less successful in that lice seem to leave such fish --- but this is merely anecdotal.

Not too sure what to recommend with regards to the model --- perhaps assume that adult females stay on the fish and die while males try to move and succeed at a certain rate? Preadult lice: I'm not aware of any data, so this would be guesswork. Females start reproducing at the late preadult II stage, so perhaps model these as the adult females, and all other preadults similar to males.

**Comment by magicicada** [comment]
Currently, lice on dead fish die - this issue is to add more nuanced behaviour

**Comment by erolm-a** [comment]
I guess for now the wished behaviour will be to make female lice die and male lice move to neighbouring fish with a parametric probability. A small issue is to see what happens when treatment has been applied, as for this period further infections cannot occur.

**Comment by anthonyohare** [comment]
If it is an easy change to make male lice detach and move that would be great, but there are a few unknowns such as the probably/rate they succeed at finding a new host.
My feeling is that it probably won't make that much of a difference so we could just leave it as it is (all lice die when the fish die) and make a note or low priority issue for the future for when we have the correct data on the success rates (which may be never!).
